charts/pega/charts/installer/config/prpcUtils.properties.tmpl

```diff
@@ -26,6 +26,10 @@ customerdata.schema.name={{ .Env.CUSTOMERDATA_SCHEMA }}
 # Will use default if not set to valid directory
 user.temp.dir=/opt/pega/temp
 
+# CUSTOM JVM ARGS
+# Any additional jvm arguments other than Max Heap Size
```

Comment:
Please update the wording to be more clear>>
Use this parameter to add jvm arguments other than Max Heap Size so your deployment nodes invoke these arguments each time the nodes start.

Assuming you expose these args in another client-facing yaml, you can re-use the text.